src/util.hpp

```diff
@@ -24,27 +24,12 @@
 #define ATTRIBUTE_RETURNS_NOALIAS __declspec(restrict)
 #define ATTRIBUTE_NORETURN __declspec(noreturn)
 
-static inline int clzll(unsigned long long mask) {
```

Comment:
Had to move the clzll definition lower in the file so it could see the `zig_unrechable` definition.